integration_tests/followme_target.cpp

```diff
@@ -0,0 +1,41 @@
+#include <iostream>
+#include <cmath>
+#include <thread>
+#include <chrono>
+#include "integration_test_helper.h"
+#include "global_include.h"
+#include "dronecore.h"
+
+using namespace dronecore;
+
+TEST_F(SitlTest, FollowMeTargetTest)
```

Comment:
No `Test` in name needed, just `FollowMe` is fine.

Reply:
Done